permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java

```diff
@@ -409,12 +409,12 @@ void requestPermissions(
                     Settings.ACTION_REQUEST_SCHEDULE_EXACT_ALARM,
                     PermissionConstants.PERMISSION_CODE_SCHEDULE_EXACT_ALARM);
             } else if (permission == PermissionConstants.PERMISSION_GROUP_CALENDAR_FULL_ACCESS || permission == PermissionConstants.PERMISSION_GROUP_CALENDAR) {
-                // Deny CALENDAR_FULL_ACCESS permission if manifest is not listing both read- and write permissions.
-                // Otherwise, we will only ask for READ permission and think full access is granted.
+                // Deny CALENDAR_FULL_ACCESS permission if manifest is not listing both write- and read permissions.
+                // Otherwise, we will only ask for WRITE permission and think full access is granted.
```

Comment:
Remove this sentence, upon first read this is a bit confusing and I think the first sentence in the comment already explains enough.